dokka-integration-tests/settings.gradle.kts

```diff
@@ -36,7 +42,7 @@ dependencyResolutionManagement {
 includeBuild("../dokka-runners/runner-gradle-plugin-classic")
 includeBuild("../dokka-runners/runner-maven-plugin")
 includeBuild("../dokka-runners/runner-cli")
-includeBuild("../.") // include the very root aggregating build so that we can depend on its tasks
```

Comment:
I removed this comment because it's no longer accurate: dokka-integration-tests depends on dependencies from the root project (the same as the other includedBuild dependencies).